src/quail/sensors/ADCTask.cpp

```diff
@@ -42,10 +44,27 @@ void ADCTask::activity()
                     sys.tasks.txtask.writeUSB(msg_buffer);
                 }
             } else { // failed ADC read, check checksum and SPI
-                err_count++;
-                char msg_buffer[50];
-                sprintf(msg_buffer, "ADC data error %i (%i time)", ret, err_count);
-                sys.tasks.txtask.writeUSB(msg_buffer);
+                uint32_t error_reg = sys.adc.getRegister(Ad7124::RegisterId::Error);
+                sys.adc.setIRQAction(adcISR);
+
+                uint32_t error_mask = AD7124_ERR_REG_ADC_CONV_ERR |
+                                      AD7124_ERR_REG_AINM_UV_ERR |
+                                      AD7124_ERR_REG_AINM_OV_ERR |
+                                      AD7124_ERR_REG_AINP_UV_ERR |
+                                      AD7124_ERR_REG_AINP_OV_ERR;
```

Comment:
I get not worrying about under/over-voltage. But what about the ADC_CONV? When does that get rigerred?

Reply:
These were all errors that got triggered regularly when a sensor was unplugged.